[🐛]: KubeJS Server errors with 0.10.11

### Was this bug reported before?

- [x] No, it was not.

### Version modpack used

0.10.11

### Environment

MP

### New Worlds

Yes

### Please provide a link to your latest.log, if relevant.

NA

### Expected Behavior

No errors

### Actual Behavior

Errors

### Steps to Reproduce

Start server

### Additional Information

[05:20:14] [ERROR] ! soulbound/recipes.js#21: Could not create ID from 'tfg:soulbind_ad_astra:space_helmet'!: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:space_helmet
[05:20:14] [ERROR] ! net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:space_helmet
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.m_245185_(ResourceLocation.java:242)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:40)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:44)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:50)
[05:20:14] [ERROR] !   at â€¦kubejs.util.UtilsJS.getMCID(UtilsJS.java:428)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaObject.coerceTypeImpl(NativeJavaObject.java:268)
[05:20:14] [ERROR] !   at â€¦rhino.Context.jsToJava(Context.java:237)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaMethod.call(NativeJavaMethod.java:430)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.iterativeMethod(NativeArray.java:1314)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.execIdCall(NativeArray.java:1798)
[05:20:14] [ERROR] !   at â€¦rhino.IdFunctionObject.call(IdFunctionObject.java:103)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.Context.doTopCall(Context.java:1374)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:70)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.InterfaceAdapter.invoke(InterfaceAdapter.java:125)
[05:20:14] [ERROR] !   at â€¦rhino.VMBridge.lambda$newInterfaceProxy$0(VMBridge.java:74)
[05:20:14] [ERROR] !   at TRANSFORMER/jdk.proxy4/jdk.proxy4.$Proxy100.onEvent(Unknown Source)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandlerContainer.handle(EventHandlerContainer.java:39)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.postToHandlers(EventHandler.java:304)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:272)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:226)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:189)
[05:20:14] [ERROR] !   at â€¦kubejs.recipe.RecipesEventJS.post(RecipesEventJS.java:359)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.handler$ccd000$customRecipesHead(RecipeManager.java:5031)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214652_(Util.java:783)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214679_(Util.java:772)
[05:20:14] [ERROR] !   at net.minecraft.server.Main.main(Main.java:166)
[05:20:14] [ERROR] !   at java.lang.reflect.Method.invoke(Method.java:569)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[05:20:14] [ERROR] !   at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
[05:20:14] [ERROR] ! soulbound/recipes.js#21: Could not create ID from 'tfg:soulbind_ad_astra:space_suit'!: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:space_suit
[05:20:14] [ERROR] ! net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:space_suit
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.m_245185_(ResourceLocation.java:242)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:40)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:44)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:50)
[05:20:14] [ERROR] !   at â€¦kubejs.util.UtilsJS.getMCID(UtilsJS.java:428)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaObject.coerceTypeImpl(NativeJavaObject.java:268)
[05:20:14] [ERROR] !   at â€¦rhino.Context.jsToJava(Context.java:237)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaMethod.call(NativeJavaMethod.java:430)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.iterativeMethod(NativeArray.java:1314)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.execIdCall(NativeArray.java:1798)
[05:20:14] [ERROR] !   at â€¦rhino.IdFunctionObject.call(IdFunctionObject.java:103)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.Context.doTopCall(Context.java:1374)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:70)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.InterfaceAdapter.invoke(InterfaceAdapter.java:125)
[05:20:14] [ERROR] !   at â€¦rhino.VMBridge.lambda$newInterfaceProxy$0(VMBridge.java:74)
[05:20:14] [ERROR] !   at TRANSFORMER/jdk.proxy4/jdk.proxy4.$Proxy100.onEvent(Unknown Source)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandlerContainer.handle(EventHandlerContainer.java:39)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.postToHandlers(EventHandler.java:304)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:272)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:226)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:189)
[05:20:14] [ERROR] !   at â€¦kubejs.recipe.RecipesEventJS.post(RecipesEventJS.java:359)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.handler$ccd000$customRecipesHead(RecipeManager.java:5031)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214652_(Util.java:783)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214679_(Util.java:772)
[05:20:14] [ERROR] !   at net.minecraft.server.Main.main(Main.java:166)
[05:20:14] [ERROR] !   at java.lang.reflect.Method.invoke(Method.java:569)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[05:20:14] [ERROR] !   at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
[05:20:14] [ERROR] ! soulbound/recipes.js#21: Could not create ID from 'tfg:soulbind_ad_astra:space_pants'!: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:space_pants
[05:20:14] [ERROR] ! net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:space_pants
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.m_245185_(ResourceLocation.java:242)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:40)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:44)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:50)
[05:20:14] [ERROR] !   at â€¦kubejs.util.UtilsJS.getMCID(UtilsJS.java:428)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaObject.coerceTypeImpl(NativeJavaObject.java:268)
[05:20:14] [ERROR] !   at â€¦rhino.Context.jsToJava(Context.java:237)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaMethod.call(NativeJavaMethod.java:430)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.iterativeMethod(NativeArray.java:1314)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.execIdCall(NativeArray.java:1798)
[05:20:14] [ERROR] !   at â€¦rhino.IdFunctionObject.call(IdFunctionObject.java:103)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.Context.doTopCall(Context.java:1374)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:70)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.InterfaceAdapter.invoke(InterfaceAdapter.java:125)
[05:20:14] [ERROR] !   at â€¦rhino.VMBridge.lambda$newInterfaceProxy$0(VMBridge.java:74)
[05:20:14] [ERROR] !   at TRANSFORMER/jdk.proxy4/jdk.proxy4.$Proxy100.onEvent(Unknown Source)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandlerContainer.handle(EventHandlerContainer.java:39)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.postToHandlers(EventHandler.java:304)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:272)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:226)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:189)
[05:20:14] [ERROR] !   at â€¦kubejs.recipe.RecipesEventJS.post(RecipesEventJS.java:359)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.handler$ccd000$customRecipesHead(RecipeManager.java:5031)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214652_(Util.java:783)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214679_(Util.java:772)
[05:20:14] [ERROR] !   at net.minecraft.server.Main.main(Main.java:166)
[05:20:14] [ERROR] !   at java.lang.reflect.Method.invoke(Method.java:569)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[05:20:14] [ERROR] !   at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
[05:20:14] [ERROR] ! soulbound/recipes.js#21: Could not create ID from 'tfg:soulbind_ad_astra:space_boots'!: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:space_boots
[05:20:14] [ERROR] ! net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:space_boots
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.m_245185_(ResourceLocation.java:242)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:40)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:44)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:50)
[05:20:14] [ERROR] !   at â€¦kubejs.util.UtilsJS.getMCID(UtilsJS.java:428)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaObject.coerceTypeImpl(NativeJavaObject.java:268)
[05:20:14] [ERROR] !   at â€¦rhino.Context.jsToJava(Context.java:237)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaMethod.call(NativeJavaMethod.java:430)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.iterativeMethod(NativeArray.java:1314)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.execIdCall(NativeArray.java:1798)
[05:20:14] [ERROR] !   at â€¦rhino.IdFunctionObject.call(IdFunctionObject.java:103)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.Context.doTopCall(Context.java:1374)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:70)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.InterfaceAdapter.invoke(InterfaceAdapter.java:125)
[05:20:14] [ERROR] !   at â€¦rhino.VMBridge.lambda$newInterfaceProxy$0(VMBridge.java:74)
[05:20:14] [ERROR] !   at TRANSFORMER/jdk.proxy4/jdk.proxy4.$Proxy100.onEvent(Unknown Source)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandlerContainer.handle(EventHandlerContainer.java:39)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.postToHandlers(EventHandler.java:304)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:272)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:226)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:189)
[05:20:14] [ERROR] !   at â€¦kubejs.recipe.RecipesEventJS.post(RecipesEventJS.java:359)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.handler$ccd000$customRecipesHead(RecipeManager.java:5031)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214652_(Util.java:783)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214679_(Util.java:772)
[05:20:14] [ERROR] !   at net.minecraft.server.Main.main(Main.java:166)
[05:20:14] [ERROR] !   at java.lang.reflect.Method.invoke(Method.java:569)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[05:20:14] [ERROR] !   at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
[05:20:14] [ERROR] ! soulbound/recipes.js#21: Could not create ID from 'tfg:soulbind_ad_astra:netherite_space_helmet'!: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:netherite_space_helmet
[05:20:14] [ERROR] ! net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:netherite_space_helmet
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.m_245185_(ResourceLocation.java:242)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:40)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:44)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:50)
[05:20:14] [ERROR] !   at â€¦kubejs.util.UtilsJS.getMCID(UtilsJS.java:428)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaObject.coerceTypeImpl(NativeJavaObject.java:268)
[05:20:14] [ERROR] !   at â€¦rhino.Context.jsToJava(Context.java:237)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaMethod.call(NativeJavaMethod.java:430)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.iterativeMethod(NativeArray.java:1314)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.execIdCall(NativeArray.java:1798)
[05:20:14] [ERROR] !   at â€¦rhino.IdFunctionObject.call(IdFunctionObject.java:103)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.Context.doTopCall(Context.java:1374)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:70)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.InterfaceAdapter.invoke(InterfaceAdapter.java:125)
[05:20:14] [ERROR] !   at â€¦rhino.VMBridge.lambda$newInterfaceProxy$0(VMBridge.java:74)
[05:20:14] [ERROR] !   at TRANSFORMER/jdk.proxy4/jdk.proxy4.$Proxy100.onEvent(Unknown Source)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandlerContainer.handle(EventHandlerContainer.java:39)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.postToHandlers(EventHandler.java:304)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:272)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:226)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:189)
[05:20:14] [ERROR] !   at â€¦kubejs.recipe.RecipesEventJS.post(RecipesEventJS.java:359)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.handler$ccd000$customRecipesHead(RecipeManager.java:5031)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214652_(Util.java:783)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214679_(Util.java:772)
[05:20:14] [ERROR] !   at net.minecraft.server.Main.main(Main.java:166)
[05:20:14] [ERROR] !   at java.lang.reflect.Method.invoke(Method.java:569)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[05:20:14] [ERROR] !   at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
[05:20:14] [ERROR] ! soulbound/recipes.js#21: Could not create ID from 'tfg:soulbind_ad_astra:netherite_space_suit'!: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:netherite_space_suit
[05:20:14] [ERROR] ! net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:netherite_space_suit
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.m_245185_(ResourceLocation.java:242)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:40)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:44)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:50)
[05:20:14] [ERROR] !   at â€¦kubejs.util.UtilsJS.getMCID(UtilsJS.java:428)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaObject.coerceTypeImpl(NativeJavaObject.java:268)
[05:20:14] [ERROR] !   at â€¦rhino.Context.jsToJava(Context.java:237)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaMethod.call(NativeJavaMethod.java:430)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.iterativeMethod(NativeArray.java:1314)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.execIdCall(NativeArray.java:1798)
[05:20:14] [ERROR] !   at â€¦rhino.IdFunctionObject.call(IdFunctionObject.java:103)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.Context.doTopCall(Context.java:1374)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:70)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.InterfaceAdapter.invoke(InterfaceAdapter.java:125)
[05:20:14] [ERROR] !   at â€¦rhino.VMBridge.lambda$newInterfaceProxy$0(VMBridge.java:74)
[05:20:14] [ERROR] !   at TRANSFORMER/jdk.proxy4/jdk.proxy4.$Proxy100.onEvent(Unknown Source)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandlerContainer.handle(EventHandlerContainer.java:39)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.postToHandlers(EventHandler.java:304)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:272)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:226)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:189)
[05:20:14] [ERROR] !   at â€¦kubejs.recipe.RecipesEventJS.post(RecipesEventJS.java:359)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.handler$ccd000$customRecipesHead(RecipeManager.java:5031)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214652_(Util.java:783)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214679_(Util.java:772)
[05:20:14] [ERROR] !   at net.minecraft.server.Main.main(Main.java:166)
[05:20:14] [ERROR] !   at java.lang.reflect.Method.invoke(Method.java:569)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[05:20:14] [ERROR] !   at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
[05:20:14] [ERROR] ! soulbound/recipes.js#21: Could not create ID from 'tfg:soulbind_ad_astra:netherite_space_pants'!: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:netherite_space_pants
[05:20:14] [ERROR] ! net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:netherite_space_pants
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.m_245185_(ResourceLocation.java:242)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:40)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:44)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:50)
[05:20:14] [ERROR] !   at â€¦kubejs.util.UtilsJS.getMCID(UtilsJS.java:428)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaObject.coerceTypeImpl(NativeJavaObject.java:268)
[05:20:14] [ERROR] !   at â€¦rhino.Context.jsToJava(Context.java:237)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaMethod.call(NativeJavaMethod.java:430)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.iterativeMethod(NativeArray.java:1314)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.execIdCall(NativeArray.java:1798)
[05:20:14] [ERROR] !   at â€¦rhino.IdFunctionObject.call(IdFunctionObject.java:103)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.Context.doTopCall(Context.java:1374)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:70)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.InterfaceAdapter.invoke(InterfaceAdapter.java:125)
[05:20:14] [ERROR] !   at â€¦rhino.VMBridge.lambda$newInterfaceProxy$0(VMBridge.java:74)
[05:20:14] [ERROR] !   at TRANSFORMER/jdk.proxy4/jdk.proxy4.$Proxy100.onEvent(Unknown Source)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandlerContainer.handle(EventHandlerContainer.java:39)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.postToHandlers(EventHandler.java:304)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:272)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:226)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:189)
[05:20:14] [ERROR] !   at â€¦kubejs.recipe.RecipesEventJS.post(RecipesEventJS.java:359)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.handler$ccd000$customRecipesHead(RecipeManager.java:5031)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214652_(Util.java:783)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214679_(Util.java:772)
[05:20:14] [ERROR] !   at net.minecraft.server.Main.main(Main.java:166)
[05:20:14] [ERROR] !   at java.lang.reflect.Method.invoke(Method.java:569)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[05:20:14] [ERROR] !   at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
[05:20:14] [ERROR] ! soulbound/recipes.js#21: Could not create ID from 'tfg:soulbind_ad_astra:netherite_space_boots'!: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:netherite_space_boots
[05:20:14] [ERROR] ! net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: tfg:soulbind_ad_astra:netherite_space_boots
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.m_245185_(ResourceLocation.java:242)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:40)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:44)
[05:20:14] [ERROR] !   at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:50)
[05:20:14] [ERROR] !   at â€¦kubejs.util.UtilsJS.getMCID(UtilsJS.java:428)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaObject.coerceTypeImpl(NativeJavaObject.java:268)
[05:20:14] [ERROR] !   at â€¦rhino.Context.jsToJava(Context.java:237)
[05:20:14] [ERROR] !   at â€¦rhino.NativeJavaMethod.call(NativeJavaMethod.java:430)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.iterativeMethod(NativeArray.java:1314)
[05:20:14] [ERROR] !   at â€¦rhino.NativeArray.execIdCall(NativeArray.java:1798)
[05:20:14] [ERROR] !   at â€¦rhino.IdFunctionObject.call(IdFunctionObject.java:103)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpretLoop(Interpreter.java:986)
[05:20:14] [ERROR] !   at â€¦rhino.Interpreter.interpret(Interpreter.java:370)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:72)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.Context.doTopCall(Context.java:1374)
[05:20:14] [ERROR] !   at â€¦rhino.InterpretedFunction.call(InterpretedFunction.java:70)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.ArrowFunction.call(ArrowFunction.java:42)
[05:20:14] [ERROR] !   at â€¦rhino.Context.callSync(Context.java:1357)
[05:20:14] [ERROR] !   at â€¦rhino.InterfaceAdapter.invoke(InterfaceAdapter.java:125)
[05:20:14] [ERROR] !   at â€¦rhino.VMBridge.lambda$newInterfaceProxy$0(VMBridge.java:74)
[05:20:14] [ERROR] !   at TRANSFORMER/jdk.proxy4/jdk.proxy4.$Proxy100.onEvent(Unknown Source)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandlerContainer.handle(EventHandlerContainer.java:39)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.postToHandlers(EventHandler.java:304)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:272)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:226)
[05:20:14] [ERROR] !   at â€¦kubejs.event.EventHandler.post(EventHandler.java:189)
[05:20:14] [ERROR] !   at â€¦kubejs.recipe.RecipesEventJS.post(RecipesEventJS.java:359)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.handler$ccd000$customRecipesHead(RecipeManager.java:5031)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java)
[05:20:14] [ERROR] !   at net.minecraft.world.item.crafting.RecipeManager.m_5787_(RecipeManager.java:34)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
[05:20:14] [ERROR] !   at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
[05:20:14] [ERROR] !   at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214652_(Util.java:783)
[05:20:14] [ERROR] !   at net.minecraft.Util.m_214679_(Util.java:772)
[05:20:14] [ERROR] !   at net.minecraft.server.Main.main(Main.java:166)
[05:20:14] [ERROR] !   at java.lang.reflect.Method.invoke(Method.java:569)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/fmlloader@1.20.1-47.4.8/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
[05:20:14] [ERROR] !   at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
[05:20:14] [ERROR] !   at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
[05:20:14] [ERROR] ! gregtech/utility.js#54: Failed to create recipe for type 'gtceu:cutter': Failed to create recipe for type 'gtceu:cutter' with args [tfg:sns:leather_strip_cut: ConsString]
[05:20:14] [WARN] gregtech/utility.js#54: Tried to call a function on an errored recipe! (Failed to create recipe for type 'gtceu:cutter')
[05:20:14] [ERROR] ! gregtech/utility.js#54: Error in 'ServerEvents.recipes': TypeError: Cannot find default value for object.

--- a/kubejs/server_scripts/soulbound/recipes.js
+++ b/kubejs/server_scripts/soulbound/recipes.js
@@ -18,7 +18,7 @@ const registerSoulboundRecipes = (event) => {
 				}
 				result.nbt.put('soulbindingSoulboundItems', true)
 				return result;
-			}).id(`tfg:soulbind_${x.equipment}`);
+			}).id(`tfg:soulbind_${x.equipment}`.replace(/[: ]/g, '_'));
 
 	})
 

